e2e/support/helpers/e2e-sample-apps-helpers.js

```diff
@@ -0,0 +1,33 @@
+function convertStringListToArray(stringList) {
+  return (stringList ?? "")
+    .split(",")
+    .map(item => item.trim())
+    .filter(Boolean);
+}
+
+export function getSampleAppsNamesFromEnv() {
```

Comment:
This helper is used  both in a cypress config and in the new runner. 
Not sure if it's the right location for such helper.